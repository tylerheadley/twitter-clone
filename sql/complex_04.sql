/*
 * Selects the text of tweets that are written in English and contain the hashtag #coronavirus 
 */
SELECT
    tweets.id_tweets,
    length(text)
FROM tweet_tags
INNER JOIN tweets ON tweet_tags.id_tweets = tweets.id_tweets
WHERE
    lang = 'en' AND
    tag = '#coronavirus' 
ORDER BY tweets.id_tweets;
