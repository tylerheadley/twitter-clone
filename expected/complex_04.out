      id_tweets      | length 
---------------------+--------
 1345158404080328705 |    111
 1345158968541265920 |    150
 1347694832328732673 |    271
 1347695181781495808 |     46
(4 rows)

